pulseapi/creators/tests.py

```diff
@@ -46,3 +50,19 @@ def test_creator_filtering(self):
 
         self.assertEqual(response.status_code, 200)
         self.assertEqual(profile.id, response_creator['creator_id'])
+
+    def test_creator_filtering_hiding_inactive_users(self):
+        """Testing that inactive users are not returned in the creator view"""
+        profile = UserProfile.objects.last()
+        # Creating an "inactive" user account to link to the profile,
+        # as we are filtering to show only active profiles in our API views.
+        BasicEmailUserFactory(profile=profile, is_active=False)
```

Comment:
**Issue** The user should be active, but the profile should be inactive. That is the change in the logic we wanted. 

I guess the docs string and method name should also be updated accordingly.